packages/core/build/contracts/IERC165.json

```diff
@@ -0,0 +1,334 @@
+{
```

Comment:
Build artifacts are usually not committed :)

Reply:
To avoid commit `/build`, add `"prepare": "you_build_script"` in `package.json` scripts

Reply:
forgot to add it to packages/code/.gitignore